install_config/topics/ocp_osp_base_image.adoc

```diff
@@ -0,0 +1,52 @@
+==== Creating a Red Hat Enterprise Linux Base Image
+
+After the user and project are created, a cloud image is required for all the
+{rhosp} instances that are to install {product-title}. For this particular reference
+environment the image used is {rhel} {rhel_version}.
+
+NOTE: {rhel} {rhel_version} includes support for overlay2 file system used in
+this reference environment for container storage purposes. If using prior
+{rhel} versions, do not use overlay2 file system.
+
+The {rhel} {rhel_version} cloud image is located at https://access.redhat.com/downloads/content/69/ver=/rhel---7/{rhel_version}/x86_64/product-software[Red Hat Enterprise Linux {rhel_version} KVM Guest Image]
```

Comment:
I don't have the right to check this URL

Reply:
The URL requires you to have access to access.redhat.com which customers should have.